### PR TITLE
Add change events to operations, by adding namespace to ResolveOperation

### PR DESCRIPTION
**Pull request by peterbroadhurst**
In PR chain with #738

### CODE REVIEW

File: smart_contracts/ethereum/solidity_firefly/artifacts/contracts/Firefly.sol/Firefly.dbg.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (awrichar) [inline smart_contracts/ethereum/solidity_firefly/artifacts/contracts/Firefly.sol/Firefly.dbg.json:1]: Was it intentional to check in the "artifacts" and "cache" folders? Seems like they should be gitignored.

File: internal/database/sqlcommon/operation_sql.go [hunk 2]
```diff
@@ -180,15 +180,21 @@ func (s *SQLCommon) UpdateOperation(ctx context.Context, id *fftypes.UUID, updat
...
 
+	for _, hook := range hooks {
+		s.postCommitEvent(tx, hook)
+	}
+
 	return s.commitTx(ctx, tx, autoCommit)
 }
 
-func (s *SQLCommon) ResolveOperation(ctx context.Context, id *fftypes.UUID, status fftypes.OpStatus, errorMsg string, output fftypes.JSONObject) (err error) {
+func (s *SQLCommon) ResolveOperation(ctx context.Context, ns string, id *fftypes.UUID, status fftypes.OpStatus, errorMsg string, output fftypes.JSONObject) (err error) {
```
Comment (awrichar) [inline internal/database/sqlcommon/operation_sql.go:190]: Feels slightly wrong that `ns` makes its way to the change event but _not_ to the update filter. Should we ensure that the operation being updated _is_ actually in this namespace?

Comment (peterbroadhurst) [reply internal/database/sqlcommon/operation_sql.go:190]: Going to address this in a new PR, from a fork, as I can't rebase this branch.

**Comment by peterbroadhurst** [comment]
Closing to create a new PR from a fork rebased

**Pull request closed without merging**